dali/test/python/test_operator_python_function.py

```diff
@@ -289,15 +295,15 @@ def invalid_function(image):
     return img
 
 
-@raises(Exception)
+@raises(RuntimeError, regex="img.*not defined")
```

Comment:
`img`? Shouldn't we print `image`?

Reply:
`img` - look at the function above - it complains about `img` being undefined.

Reply:
Got it.